pkg/mod/github.go

```diff
@@ -69,8 +69,12 @@ func (d *githubMgr) Get(filename, ver string, m *Modules) (*Module, error) {
 	if err != nil {
 		return nil, err
 	}
-	if ver == "" {
-		ver = "v0.0.0-" + fileContent.GetSHA()[:12]
+	if ver == "" || ver == MasterBranch {
+		ref, _, err := d.client.Git.GetRef(ctx, repoPath.owner, repoPath.repo, "heads/"+MasterBranch)
+		if err != nil {
+			return nil, err
+		}
+		ver = "v0.0.0-" + ref.GetObject().GetSHA()[:12]
```

Comment:
Also short commit hashes are usually 7 digits (didn't know that off the top of my head) https://www.shellhacks.com/git-get-short-hash-sha-1-from-long-hash-head-log/

Is there any reason why we're grabbing the first 12?

Reply:
Cuz go modules use first 12

Reply:
Ahhh I see! No worries sounds good